advanced_alchemy/service/_async.py

```diff
@@ -316,6 +317,11 @@ async def update(
         data = await self.to_model(data, "update")
         if isinstance(id_attribute, InstrumentedAttribute):
             id_attribute = cast("str", id_attribute.description)
+        if item_id is None and self.repository.get_id_attribute_value(item=data, id_attribute=id_attribute) is None:
+            msg = f"Could not identify ID attribute value.  One of the following is required: ``item_id`` or ``data.{id_attribute or self.repository.id_attribute}``"
```

Comment:
```suggestion
            msg = (
                "Could not identify ID attribute value.  One of the following is required: "
                f"``item_id`` or ``data.{id_attribute or self.repository.id_attribute}``"
            )
```

Reply:
The way GH wraps these lines makes these edits look horrendous, but its just breaking the string up over two lines.